### PR TITLE
#3503 correct presentation server license

### PR DESCRIPTION
**Pull request by davidradl**
Signed-off-by: David Radley <david_radley@uk.ibm.com>

### CODE REVIEW

**Comment by davidradl** [comment]
@planetf1 Are you ok with this change?

File: open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "dev": "webpack-dev-server --open"
   },
   "author": "David Radley",
-  "license": "ISC",
+  "license": "Apache-2",
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json:18]: I think it should be 'Apache-2.0' - but just checkign ...

Comment (planetf1) [reply open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json:18]: Yes - so needs updating
ref: https://spdx.org/licenses/

**Review by planetf1 — CHANGES_REQUESTED** [review]
License needs updating to 'Apache-2.0'

File: open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json (hunk 1 above)
Comment (davidradl) [reply open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json]: sorry I should have checked - I assumed the sting in the git issue text was correct. Fixed.

**Review by planetf1 — APPROVED** [review]

**Comment by planetf1** [comment]
All looks good

**Pull request merged**